app/components/questions/BelongingRates.js

```diff
@@ -57,6 +60,48 @@ function transformData(response, error, selectedArea) {
 
 function ExamplePieChart(props) {
   const { data } = props
+  
+  
+  const tooltipStyle = {
+    backgroundColor: "white", 
+    borderColor: "LightGrey", 
+    borderStyle: "solid", 
+    borderWidth: "thin"
+  };
+  
+  
+  const CustomTooltip = ({ active, payload, label }) => {
+  if (active && payload && payload.length) {
+    // Changes the tooltip to display the formatted percentage rather than a decimal
+    return (
+      <div className="custom-tooltip" style={tooltipStyle}>
+        <p className="label">{`${payload[0].name} : ${(payload[0].value * 100).toFixed(1)}%`}</p>
```

Comment:
Instead of:

```js
${(payload[0].value * 100).toFixed(1)}%
```

Use:

```js
${payload[0].valueFormatted}
```

Reply:
Confusingly, this change doesn't work as expected. I might have to look into the way the payload works, but on the page it displays as `undefined` instead of what you'd expect from `valueFormatted`

![image](https://user-images.githubusercontent.com/72165627/126387100-aac515cc-3f2e-45d8-9a8e-6cd7de06cca4.png)

Reply:
Oh, haha I logged `payload` and it looks like there's another level to go into:

![Screen Shot 2021-07-20 at 3 06 33 PM](https://user-images.githubusercontent.com/11896652/126388653-bd2a5e3d-e60d-4f2b-aa1c-14230f71dc57.png)

So we can do:

```js
const entry = payload[0].payload;
...
${payload.valueFormatted}
```

Reply:
I think I've figured it out. Instead of
```javascript
${payload[0].valueFormatted}
```
we need to use 
```javascript
${payload[0].payload.valueFormatted}
```
This is a bit weird but it works from what I can see.